WordPress/Classes/Extensions/UIScrollView+Helpers.swift

```diff
@@ -15,3 +15,9 @@ extension UIScrollView {
         }
     }
 }
+
+extension UICollectionView {
+    @objc override func scrollToTop(animated: Bool) {
```

Comment:
Note: `scrollToTop` is a method [introduced by the app](https://github.com/wordpress-mobile/WordPress-iOS/blob/57ce3143ac9c79582ff1ca0f8f39e696d4f6d193/WordPress/Classes/Extensions/UIScrollView%2BHelpers.swift#L4), so here it overrides it using subclassing and Objective-C runtime.

It's odd that is has `layoutIfNeeded` in there too, but that's unrelated:

```
extension UIScrollView {
    @objc func scrollToTop(animated: Bool) {
        let topOffset = CGPoint(x: 0, y: -adjustedContentInset.top)
        setContentOffset(topOffset, animated: animated)
        layoutIfNeeded()
    }
```